bf2fc6cc711aee1a0c2a/cos-observability-resources
# MGDCTRS-1757: Modify midstream rules accordingly to cluster_id

## Description

- alerts will be running in Observatorium, it is necessary to group by cluster_id now
- some formatting of summary and description
- change summary to a less wordy sentence
- change observatorium workflow token from my personal one to github's automatically generated one
## Verification Steps

1. Since all modified checks are similar, testing one of them in Observatorium Stage should be enough. For this testing exercise we use `CosFleetShardSyncTargetDown`. Pick one cluster you have access to and that send metrics to Observatorium RHOC Staging tenant (dev cluster for instance).
2. Stop the cos-fleetshard-sync pod in dev, wait 10 minutes.
3. Run the following query (you have to setup obsctl and be logged onto rhoc staging tenant):
```
obsctl metrics query 'group by (cluster_id) (up{container="cos-fleetshard-sync"} !=1 or absent(up{container="cos-fleetshard-sync"}))'

{
        "status": "success",
        "data": {
                "resultType": "vector",
                "result": [
                        {
                                "metric": {
                                        "cluster_id": "~~redacted~~"
                                },
                                "value": [
                                        1670862046.781,
                                        "1"
                                ]
                        }
                ]
        }
}
```
4. The resulting cluster_id should match the cluster_id of the cluster you are using for testing.

## Checklist (Definition of Done)

~~Required Standard Operating Procedure (SOP) is added.~~

## Prometheus

- [x] Changes were tested against dev environment

## Grafana

~~You have imported the changes into a dev or staging environment~~
